### ISSUE TITLE
Entering a number from keyboard in Publish - Schedule date is causing the dialog to shown an error.

### ISSUE THREAD

**Issue by yacdaniel** [craftercms/craftercms#7495]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] The issue is in the develop branch
- [ ] The issue is in the latest released 4.1.x

### Describe the issue

When an user schedule content to publish later and if the user types a number in any of the Date-Time fields, the Date will shown an "Invalid Date" error message, the selected date-time stays the same without change and the error message won't go away.

Entering an number should change the Date-TIme value or don't change anything but without reporting an error. 

### Steps to reproduce

Steps:

1. Create a project from any BP, do the Initial Publish
2. Righ click content item and select `Publish` then in Scheduling mark `Later`
3. Type a number in the Date or Time fields. 
4. See the issue

### Relevant log output

```shell
tatic-assets/app/as…te-4xRiCTiM.js:2257 Uncaught TypeError: Cannot read properties of null (reading '1')
    at get8601Pieces (static-assets/app/as…p3ANdBO.js:27211:17)
    at static-assets/app/as…CNZGpfhU.js:1879:32
    at static-assets/app/as…BYPhxqEk.js:1883:33
    at static-assets/app/as…xRiCTiM.js:10197:22
    at static-assets/app/as…-BYPhxqEk.js:7900:7
    at static-assets/app/as…xRiCTiM.js:10197:22
    at static-assets/app/as…BYPhxqEk.js:7974:73
    at static-assets/app/as…xRiCTiM.js:10197:22
    at static-assets/app/as…BYPhxqEk.js:1883:33
    at static-assets/app/as…xRiCTiM.js:10197:22
```

### Screenshots and/or videos

https://github.com/user-attachments/assets/00965de7-2a8a-4b9e-9b60-4e8701c65017

**Comment by Euquimides** [comment]
Hi @rart and @jvega190 ! While validating this ticket I noticed that pressing backspace inside the Publish-Schedule date dialog, will throw an error in the date field as shown in the attached video. The following error shows up in the browser's console:

```
09:19:42.169 Content-Security-Policy: This site (http://localhost:8080) has a Report-Only policy without a report-uri directive nor a report-to directive. CSP will not block and cannot report violations of this policy. preview
09:19:45.923 Cookie “JSESSIONID” does not have a proper “SameSite” attribute value. Soon, cookies without the “SameSite” attribute or with an invalid value will be treated as “Lax”. This means that the cookie will no longer be sent in third-party contexts. If your application depends on this cookie being available in such contexts, please add the “SameSite=None“ attribute to it. To know more about the “SameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite localhost:8080
09:19:45.925 Content-Security-Policy: This site (http://localhost:8080) has a Report-Only policy without a report-uri directive nor a report-to directive. CSP will not block and cannot report violations of this policy. localhost:8080
09:27:07.867 Uncaught TypeError: pieces is null
    get8601Pieces http://localhost:8080/studio/static-assets/app/assets/index-j2n2KBai.js:27211
    createOnDateChange http://localhost:8080/studio/static-assets/app/assets/index-RzggfYqm.js:1883
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    updateDate http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7900
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChangeFromField http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7974
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    publishValue http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2024
    clearActiveSection http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2052
    handleInputChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2994
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChange http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:24946
    Nb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1277
    Tb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1289
    Ub http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1292
    nf http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2229
    se$1 http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2252
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2524
    Qk http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:5828
    Jb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1230
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2325
    fd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1723
    ed http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1706
3 index-j2n2KBai.js:27211:11
09:27:51.494 Uncaught TypeError: pieces is null
    get8601Pieces http://localhost:8080/studio/static-assets/app/assets/index-j2n2KBai.js:27211
    createOnDateChange http://localhost:8080/studio/static-assets/app/assets/index-RzggfYqm.js:1883
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    updateDate http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7900
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChangeFromField http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7974
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    publishValue http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2024
    clearActiveSection http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2052
    handleInputChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2994
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChange http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:24946
    Nb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1277
    Tb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1289
    Ub http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1292
    nf http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2229
    se$1 http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2252
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2524
    Qk http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:5828
    Jb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1230
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2325
    fd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1723
    ed http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1706
2 index-j2n2KBai.js:27211:11
09:30:25.292 Uncaught TypeError: pieces is null
    get8601Pieces http://localhost:8080/studio/static-assets/app/assets/index-j2n2KBai.js:27211
    createOnDateChange http://localhost:8080/studio/static-assets/app/assets/index-RzggfYqm.js:1883
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    updateDate http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7900
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChangeFromField http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7974
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    publishValue http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2024
    clearActiveSection http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2052
    handleInputChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2994
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChange http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:24946
    Nb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1277
    Tb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1289
    Ub http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1292
    nf http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2229
    se$1 http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2252
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2524
    Qk http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:5828
    Jb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1230
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2325
    fd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1723
    ed http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1706
index-j2n2KBai.js:27211:11
09:30:29.730 Uncaught TypeError: newMoment is null
    createOnDateChange http://localhost:8080/studio/static-assets/app/assets/index-RzggfYqm.js:1870
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    updateDate http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7900
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChangeFromField http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7974
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    publishValue http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2024
    clearActiveSection http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2052
    handleInputChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2994
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChange http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:24946
    Nb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1277
    Tb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1289
    Ub http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1292
    nf http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2229
    se$1 http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2252
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2524
    Qk http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:5828
    Jb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1230
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2325
    fd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1723
    ed http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1706
index-RzggfYqm.js:1870:21
09:30:55.052 Uncaught TypeError: pieces is null
    get8601Pieces http://localhost:8080/studio/static-assets/app/assets/index-j2n2KBai.js:27211
    createOnDateChange http://localhost:8080/studio/static-assets/app/assets/index-RzggfYqm.js:1883
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    updateDate http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7900
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChangeFromField http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7974
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    publishValue http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2024
    clearActiveSection http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2052
    handleInputChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2994
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChange http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:24946
    Nb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1277
    Tb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1289
    Ub http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1292
    nf http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2229
    se$1 http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2252
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2524
    Qk http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:5828
    Jb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1230
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2325
    fd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1723
    ed http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1706
index-j2n2KBai.js:27211:11
09:30:58.348 Uncaught TypeError: newMoment is null
    createOnDateChange http://localhost:8080/studio/static-assets/app/assets/index-RzggfYqm.js:1870
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    updateDate http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7900
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChangeFromField http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7974
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    publishValue http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2024
    clearActiveSection http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2052
    handleInputChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2994
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChange http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:24946
    Nb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1277
    Tb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1289
    Ub http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1292
    nf http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2229
    se$1 http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2252
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2524
    Qk http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:5828
    Jb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1230
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2325
    fd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1723
    ed http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1706
index-RzggfYqm.js:1870:21
09:31:46.100 Uncaught TypeError: pieces is null
    get8601Pieces http://localhost:8080/studio/static-assets/app/assets/index-j2n2KBai.js:27211
    createOnDateChange http://localhost:8080/studio/static-assets/app/assets/index-RzggfYqm.js:1883
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    updateDate http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7900
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChangeFromField http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7974
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    publishValue http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2024
    clearActiveSection http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2052
    handleInputChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2994
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChange http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:24946
    Nb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1277
    Tb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1289
    Ub http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1292
    nf http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2229
    se$1 http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2252
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2524
    Qk http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:5828
    Jb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1230
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2325
    fd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1723
    ed http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1706
2 index-j2n2KBai.js:27211:11
09:32:30.408 Uncaught TypeError: pieces is null
    get8601Pieces http://localhost:8080/studio/static-assets/app/assets/index-j2n2KBai.js:27211
    createOnDateChange http://localhost:8080/studio/static-assets/app/assets/index-RzggfYqm.js:1883
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    updateDate http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7900
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChangeFromField http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7974
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    publishValue http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2024
    clearActiveSection http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2052
    handleInputChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2994
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChange http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:24946
    Nb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1277
    Tb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1289
    Ub http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1292
    nf http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2229
    se$1 http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2252
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2524
    Qk http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:5828
    Jb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1230
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2325
    fd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1723
    ed http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1706
index-j2n2KBai.js:27211:11
09:59:01.340 Uncaught TypeError: pieces is null
    get8601Pieces http://localhost:8080/studio/static-assets/app/assets/index-j2n2KBai.js:27211
    createOnDateChange http://localhost:8080/studio/static-assets/app/assets/index-RzggfYqm.js:1883
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    updateDate http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7900
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChangeFromField http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7974
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    publishValue http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2024
    clearActiveSection http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2052
    handleInputChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2994
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChange http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:24946
    Nb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1277
    Tb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1289
    Ub http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1292
    nf http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2229
    se$1 http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2252
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2524
    Qk http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:5828
    Jb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1230
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2325
    fd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1723
    ed http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1706
index-j2n2KBai.js:27211:11
09:59:03.824 Uncaught TypeError: newMoment is null
    createOnDateChange http://localhost:8080/studio/static-assets/app/assets/index-RzggfYqm.js:1870
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    updateDate http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7900
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChangeFromField http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:7974
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleValueChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:1883
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    publishValue http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2024
    clearActiveSection http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2052
    handleInputChange http://localhost:8080/studio/static-assets/app/assets/useMobilePicker-BHHWOzRL.js:2994
    useEventCallback http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:10197
    handleChange http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:24946
    Nb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1277
    Tb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1289
    Ub http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1292
    nf http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2229
    se$1 http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2252
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2524
    Qk http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:5828
    Jb http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1230
    hd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:2325
    fd http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1723
    ed http://localhost:8080/studio/static-assets/app/assets/ErrorState-BBOmvZNl.js:1706
index-RzggfYqm.js:1870:21
```

Can you check this out? Thanks!

https://github.com/user-attachments/assets/7b3ecb5e-4649-45ac-b008-f2340fc96527

**Comment by sumerjabri** [comment]
@Euquimides please create another enhancement ticket to keep the console clean. Please validate this ticket.

**Comment by Euquimides** [comment]
Hi! Updated and rechecked this issue. Fix verified for v4.2.0. Also, added an enhancement request related with this ticket in https://github.com/craftercms/craftercms/issues/7501. Thank you! Closing this ticket.
